rust/private/rustc.bzl

```diff
@@ -395,6 +409,16 @@ def collect_inputs(
     return _process_build_scripts(ctx, file, crate_info, build_info, dep_info, compile_inputs)
 
 def _exec_root_relative_path(ctx, filepath):
+    """[summary]
```

Comment:
@sitaktif Hey, can you add suggestions to this PR to fill out this documentation? 🙏

Reply:
Sorry I was a bit late to respond, I see that @illicitonion beat me to it (thanks!).